src/components/booklist.css

```diff
@@ -0,0 +1,182 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700;800&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Roboto+Slab:wght@200;300;400;500&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@200;300;400;500;600;700&display=swap');
+
+.cover-div {
+  background-color: #f5f6fa;
+  margin: 0 auto;
+  padding: 30px 50px;
+
+  .list-div {
+    list-style: none;
+
+    .list {
+      position: relative;
+      padding: 20px 0 10px 30px;
+      margin-bottom: 10px;
+      background-color: #fff;
+      border-radius: 4px;
+      border: 1px solid #e8e8e8;
+
+      .details {
+        display: flex;
+        flex-direction: column;
+
+        .text-1 {
+          opacity: 0.5;
+          font-family: 'Montserrat', Arial, Helvetica, sans-serif;
+          font-size: 14px;
+          font-weight: bold;
+          color: #121212;
+          letter-spacing: 1.5px;
+        }
+
+        .text-2 {
+          font-family: 'RobotoSlab', Arial, Helvetica, sans-serif;
```

Comment:
The previous reviewer mentioned that you weren't using the proper font families for the book cards. You did import the proper ones, and used them in your styling, it's just that you made a typo and that's why the proper font family is not being rendered. Like here, you're supposed to type `'Roboto Slab'`, there's a space in there that's missing :+1: You should update the rest of the declarations where you used `Roboto Slab`, and make sure the other ones are written correctly as well :smiley: Good luck :+1: